### DISCUSSION TITLE
How can I check my farm if is working corrctly

### DISCUSSION THREAD

**Discussion by leonneonn** [Chia-Network/chia-blockchain#1680] [Q&A]
How can I check my farm if is working correctly. I have 6 blocks and I didn't see any block pass a filter. Im afraid that my farm maybe not working correctly. I understand is hard to farm block, but this what I understand my ploots should pas filter from time to time. 
1How I can check if all is ok?
 2how I can check history and see if my plots passed filter in a past?

**Comment by Teledyol** [comment]
> How can I check my farm if is working correctly … ?

Your farming status window says "Farming," on the GUI. 

You can check your log for errors, if you really haven't seen any for the past 24 hours, you "should" be OK.

As for watching that Attempted Proof window, with 6 plots, you could be staring at it for hours and not see one attempt. There are better things to do with your time. If you're synched and the status shows that you're farming, that's the basic GUI information.

I have 49 plots (and counting) and did not see any XCH farmed until yesterday, I now have 4 XCH. That makes me better than the average of the 9 days showing on my Expected Time window, since I started with four plots in the beginning (Mar 24).

There's, likely, some Command Line scenario that will provide more information than the GUI, but my perception is that's what you have to look at, for now. With 49 plots I see a "1/49" scroll by about once every 15 minutes. I've never seen a 2/49 or two "Attempts" put of the five displayed.

**Reply by leonneonn** [reply]
Are you checking by command line? Can you wrote this comand line here so i can check too? This is what I was thinking abiut if i can look and see if some of my ploots passing atleast one filter. Then I know something is working. If im putting new plot program cant see it until I will close it and open again. Often this is happening. Thats why Im thinking to check this. If I will not see any passing filters in long long time like 24h  then maybe I need to restart program. This is what is comming to my head.
